### ISSUE TITLE
search by protein position

### ISSUE THREAD

**Issue by reimand0** [reimandlab/ActiveDriverDB#124]
When you type in a position and hit enter, the following page is given

https://activedriverdb.org/search/mutations?mutations=TP53+R282W

however when you search and click in the dropdown, then you get to the mutation page as expected

https://activedriverdb.org/mutation/show/NM_000546/282/W

I think the 2nd result should also occur in the first case. What do you think?

**Comment by reimand0** [comment]
I actually like this page a lot - https://activedriverdb.org/mutation/show/NM_000546/282/W

1. is it worth adding the list of all PTM mutations in this PTM site?
2. the dropdown menu 'show in protein context' has some issues: why only three choices, where is 1000 genomes? also, the descriptions should be more informative, like Cancer (TCGA MC3), Disease (Clinvar), population .. .

**Comment by krassowski** [comment]
The first action is interpreted like "search more" and the second as "show me this result". I see that selecting the first option could be more intuitive.

**Comment by krassowski** [comment]
> the dropdown menu 'show in protein context' has some issues: why only three choices, where is 1000 genomes? also, the descriptions should be more informative, like Cancer (TCGA MC3), Disease (Clinvar), population .. .

I will change the descriptions. There are only three choices because the mutation occurs only in three datasets. The choices adopts automatically to datasets in which the mutation occurs (including user-uploaded datasets). The link point to a specific mutation in table. Where it should point to if a mutation is not in the selected dataset?

**Comment by reimand0** [comment]
Perhaps we can show a greyed out option with '0 mutations' if no mutations
in that dataset appear?

**Comment by krassowski** [comment]
1. I added datasets that do not include the mutation with `(absent)` note, as greyed out, disabled options.

2. The datasets labels use more informative dataset names now.

3. I added mutations table to every site. I wonder if it would not confuse user who visits this page looking for information about specific mutation and finds a lots of mutation tables. On other hand it may be useful if someone wants to explore several mutations affecting given site.
